src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPCompletionFeature.java

```diff
@@ -199,7 +200,7 @@ public boolean isStrikeout(@NotNull CompletionItem item) {
     @Nullable
     public String getTailText(@NotNull CompletionItem item) {
         var labelDetails = item.getLabelDetails();
-        return labelDetails != null ? labelDetails.getDetail() : null;
+        return labelDetails != null ? labelDetails.getDetail() : item.getDetail();

```

Comment:
Please keep the original code like:

`return labelDetails != null ? labelDetails.getDetail() : null;`

Reply:
Sorry it is not resolved, please keep the original code:

`return labelDetails != null ? labelDetails.getDetail() : null;`